task/bq2bq/executor/samples/tasks/replace_by_spillover_date_query/query.sql

```diff
@@ -0,0 +1,6 @@
+select
+    TIMESTAMP('__dstart__') as dstart,
+    TIMESTAMP('__dend__') as dend,
```

Comment:
these `dstart` `dend` macros won't work now and these will be exact timestamps getting passed to this SQL, we know that right?